_tuning-your-cluster/availability-and-recovery/snapshots/sm-api.md

```diff
@@ -197,7 +197,7 @@ Parameter | Type | Description
 `deletion.delete_condition` | Object | Conditions for snapshot deletion. Optional. 
 `deletion.delete_condition.max_count` | Integer | The maximum number of snapshots to be retained. Optional.
 `deletion.delete_condition.max_age` | String | The maximum time a snapshot is retained. Optional.
-`deletion.delete_condition.min_count` | Integer | The minimum number of snapshots to be retained. Optional. Default is one.
+`deletion.delete_condition.min_count` | Integer | The minimum number of snapshots to be retained. Optional. Default is 1.
```

Comment:
```suggestion
`deletion.delete_condition.min_count` | Integer | The minimum number of snapshots to be retained. Optional. Default is `1`.
```

Reply:
There are too many of these numbers to fix that don't consistently quote them. Can be done in a separate PR.